auth-gateway/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "auth-gateway",
```

Comment:
Why is this not `authenticate` or `authenticator`? "auth" seems ambiguous between authentication and authorization, is that intentional?

Reply:
I used "gateway" in the sense that it serves at the public/private network boundary. If you'd like to choose a different name, go for it.

Reply:
I've seen people use authn/authz, I like that.

Reply:
@danking @cseed : Should we choose authz? "authz" or "authz-gateway".

Reply:
I'd probably go with authz.

Naming, the hardest part of programming.